web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NeighborInfo.java

```diff
@@ -32,6 +32,31 @@
     @JsonProperty("port_ip")
     private String portIp;
```

Comment:
For the near future, we need to support multiple IP for one port, and the IP can come from different subnet. We will need that  information for each port and each neighbor, please see the contract:
https://github.com/er1cthe0ne/alcor/blob/da2021a443444046a122f6b06c3efaad70087f2e/schema/proto3/neighbor.proto

https://github.com/er1cthe0ne/alcor/blob/da2021a443444046a122f6b06c3efaad70087f2e/schema/proto3/port.proto

relevant session below:

    message FixedIp {
        string subnet_id = 1;
        string ip_address = 2;
    }

    repeated FixedIp fixed_ips = 13;

Reply:
In the case of a port with two ip addresses, if we can think of it as two neighbors?

Reply:
That would duplicate many fields when look at the contract between DPM<->ACA. Since the two ip addresses belong to the same port, with the same port ID, I think it is better to identify it as one neighbor with two ip addresses.

Reply:
Two addresses of the port belong to two subnet, one subnet connect to a router and the other subnet does not connect to any router, is it a L3 neighbor or a L2 neighbor?

Reply:
Great point :) Let me think more about it...

Reply:
so what's the answer for this , if I would like to give sample I need this to be clear @er1cthe0ne

Reply:
support multiple IP (from different subnet) is actually a feature on its own, that include:
1) how do we provision a VM to accept multiple IPs (automatic/script/manual), note that DHCP doesn't support multiple IPs
2) need to close down on the primary/secondary IP concept
3) neighbor - do we consider a neighbor port with two IP as two different neighbor? 
4) routing - we need to make all the IPs routable, it gets more interesting when the source port has N IPs and destination port has M IPs, their will be N x M routes!

While we want to support multiple IPs per port, but need to close down on the above and I believe it is lower priority compare to our current 930 goals.